packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -186,12 +188,14 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
   }
 
   let hasAnyChildren = useMemo(() => [...collection].some(item => item.hasChildNodes), [collection]);
+  let [isFocusWithin, setIsFocusWithin] = useState(false);
+  let {focusWithinProps} = useFocusWithin({onFocusWithinChange: setIsFocusWithin});
 
   return (
-    <ListViewContext.Provider value={{state, dragState, isListDraggable, layout}}>
+    <ListViewContext.Provider value={{state, dragState, isListDraggable, layout, isFocusWithin}}>
```

Comment:
I think `selectionManager.isFocused` should already have this info?

Reply:
odd, i tried that, but i must have had a bad combo with something else, as it appears to be working when i change that to use selectionManager